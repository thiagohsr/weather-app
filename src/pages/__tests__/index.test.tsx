import { render, screen } from '@testing-library/react'
import Home from '@pages/index'

import themingWrapper from '@helpers/themingHelper';

describe('Home', () => {
  it('renders homepage unchanged', () => {
    const { container } = render(themingWrapper(<Home />))
    expect(container).toMatchSnapshot();
  })

  it('should renders a heading', () => {
    render(themingWrapper(<Home />));
    const heading = screen.getByRole('heading', {
      name: /welcome to next\.js!/i,
    })

    expect(heading).toBeInTheDocument();
  })
});
