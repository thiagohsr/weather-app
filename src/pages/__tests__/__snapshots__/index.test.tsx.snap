// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home renders homepage unchanged 1`] = `
<div>
  <div
    class="sc-a5a617a-0 fArwKn"
  >
    <main
      class="sc-a5a617a-1 gREgY"
    >
      <h1
        class="sc-a5a617a-2 isicKt"
      >
        Welcome to 
        <a
          href="https://nextjs.org"
        >
          Next.js!
        </a>
      </h1>
      <p
        class="sc-a5a617a-3 iWSRVm"
      >
        Get started by editing
        <code
          class="sc-a5a617a-4 kntbaG"
        >
          pages/index.tsx
        </code>
      </p>
    </main>
  </div>
</div>
`;
